corehq/apps/data_dictionary/bulk.py

```diff
@@ -103,7 +103,12 @@ def _process_multichoice_sheets(domain, workbook, allowed_value_info, prop_row_i
             if len(row) < 1:
                 continue
 
-            row_vals = map_row_values_to_column_names(row, column_headings, default_val='')
+            row_vals, row_val_errors = map_row_values_to_column_names(
```

Comment:
I was wondering if the errors would be repeated for all the rows if a specific column header is absent ?

Reply:
This wouldn't. This is in a loop that processes one row at a time. If there are any `row_val_errors` for a row then the entire loop will be exited with the relevant errors.

Reply:
Great. Thanks for the clarification.